### DISCUSSION TITLE
Does this app really work

### DISCUSSION THREAD

**Discussion by Valmach** [snipe/snipe-it#10281] [Q&A]
Redirect, looping sign-up,  not possible to view 'DEMO'.  Not Possible to DL.  Heroku's launch also does not work.  From IDE not working. Is there a trick to all of this.

**Comment by fabricat** [comment]
Both the [demo](https://snipeitapp.com/demo) and the `develop` demo are working ok: I can login and do things.

Or if you feel comfortable with Docker Compose, you can leverage [docker-compose.yml](https://github.com/snipe/snipe-it/blob/master/docker-compose.yml) to test it on premise 😉
